build.gradle

```diff
@@ -29,13 +29,13 @@ buildscript {
 
   dependencies {
     // For Apache Rat plugin to ignore non-Git files
-    classpath "org.ajoberstar:grgit:1.9.3"
+    classpath "org.ajoberstar.grgit:grgit-gradle:3.0.0"
```

Comment:
is the artifactId change due to incompatible upstream changes? just wondering since this is the outlier (other stuff just changes version).

Reply:
Correct, there are some breaking changes (that include artifact name change):
 * http://ajoberstar.org/grgit/index.html
 * https://github.com/ajoberstar/grgit/releases/tag/2.0.0
 * https://github.com/ajoberstar/grgit/releases/tag/3.0.0
```
As of Grgit 3, the coordinates for the library have changed.
    Instead of:
        org.ajoberstar:grgit:<version>
    Use one of:
        org.ajoberstar.grgit:grgit-core:<version>
        org.ajoberstar.grgit:grgit-gradle:<version>
The Gradle plugin ID has not changed. It is still org.ajoberstar.grgit.
```

Reply:
Resolving conversation (as per previous comment)